src/requireConfig.ts

```diff
@@ -1,11 +1,18 @@
 import { FirebaseError } from "./error";
 import { Options } from "./options";
 
+/**
+ * Rejects if there is no config in `options`.
+ */
 export async function requireConfig(options: Options): Promise<void> {
```

Comment:
nit: no longer needs to be marked async since there's no await

Reply:
but, but, it's so nice to keep it there! doesn't let it _not_ return a Promise by accident too (even though the return type there helps)